client/src/pages/Home/useStyles.ts

```diff
@@ -0,0 +1,50 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+    marginLeft: theme.spacing(15),
+  },
+  imageGrid: {
+    backgroundColor: 'red',
+    height: '100vh',
+  },
+  imageInfo: {
+    objectFit: 'cover',
+    width: '100%',
+    height: '100vh',
+  },
+  dogLabel: {
+    width: '170px',
```

Comment:
I think it would be better to use relative units for responsiveness

Reply:
Done the changes.